src/screens/Game/GameSubMenu/index.tsx

```diff
@@ -140,6 +140,12 @@ export default function GamesSubmenu({
           >
             {t('submenu.log')}
           </span>
+          <span
+            onClick={() => ipcRenderer.send('addShortcut', appName)}
+            className="hidden link"
+          >
+            {t`submenu.addShortcut`}
```

Comment:
this doesnt look right, should be:
{`${t('submenu.addShortcur', 'Add Shortcut to Desktop')}`}

Besides the function call, its good always to have a default.
After adding new i18n keys, is also good to run:
`yarn i18n` on the terminal, otherwise this new key wont be generated and weblate wont recognize it.

Reply:
Ok